geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/state/ProtobufConnectionAuthorizingStateProcessor.java

```diff
@@ -30,45 +30,18 @@
 
 public class ProtobufConnectionAuthorizingStateProcessor
     implements ProtobufConnectionStateProcessor {
-  private final SecurityService securityService;
-  private final Subject subject;
 
-  public ProtobufConnectionAuthorizingStateProcessor(SecurityService securityService,
-      Subject subject) {
-    this.securityService = securityService;
-    this.subject = subject;
-  }
+  public ProtobufConnectionAuthorizingStateProcessor() {}
 
   @Override
   public void validateOperation(Object message, ProtobufSerializationService serializer,
```

Comment:
I think this function might no longer have any point.  This was also the main driver for the creation of these state objects, so there's probably a less convoluted way of doing the remaining authentication work.  Maybe just create a task to refactor the state code?

Reply:
I'll create a task to refactor the state code separately.